### ISSUE TITLE
[Pretty Minor] Issue with electrum apps when running with proxychains4

### ISSUE THREAD

**Issue by 0XC7R** [rofl0r/proxychains-ng#586]
Config and other information has not changed since my last issue post which is here: https://github.com/rofl0r/proxychains-ng/issues/585
 
_**I will add new config and other edits if needed in the future.**_

**Not sure** if it is either electrum or proxychains related issue. **When running an app that uses electrum** like discord, signal etc you **will be required to add `--no-sandbox --in-process-gpu` at the end of the target** if you want to route its traffic through proxychains and have it boot successfully.
If you do not parse the workaround above then the process will just throw an error and exit which is supplied at the bottom and in the screenshot below

Although this work around does not always work with some applications like discord as I had to flatpak install it for it to work compared to the debian install.

I will provide a screenshot below to help show the issue:
![image](https://github.com/user-attachments/assets/ce920abe-0ed6-4466-b136-fec1cc81dfb6)
The second time I had run signal with proxychains and also parsing the workaround mentioned above is a successful start and run. The above is pretty obvious.

Console output in event of error:
```bash
┌─╼[~]
 └╼ ❯❯❯ proxychains4 signal-desktop                              
[proxychains] DLL init: proxychains-ng 4.17
[proxychains] config file found: /etc/proxychains.conf
[proxychains] preloading /usr/lib/x86_64-linux-gnu/libproxychains.so.4
[proxychains] DLL init: proxychains-ng 4.17
[proxychains] DLL init: proxychains-ng 4.17
[proxychains] DLL init: proxychains-ng 4.17
[12627:1113/182953.017777:FATAL:zygote_host_impl_linux.cc(221)] Check failed: . : Invalid argument (22)
[1]    12627 trace trap  proxychains4 signal-desktop
```

**Comment by 0XC7R** [comment]
I don't think it really has much to do with proxychains however I thought I would mention it just in case including anyone who may encounter this in the future.

**Comment by rofl0r** [comment]
that's a known issue - search for sandbox in the issues. though it's nice that you document a workaround, which may be useful for some.

**Comment by 0XC7R** [comment]
> that's a known issue - search for sandbox in the issues. though it's nice that you document a workaround, which may be useful for some.

Interesting. Kind of a flaw but is nothing really.

The work around doesn't always work for some apps like I said which means you would need to find an additional work around if you cannot use flatpak to install the same application. 
Theyre the only two workarounds I have really found. I have no clue why the deb install of apps like discord works differently compared to flatpak installation.

**Comment by 0XC7R** [comment]
Up to you but you could add this issue to the read me or I can make a pull request to add it.

**Comment by rofl0r** [comment]
i'll probably add some detail to the README next time i work on it. though it seems most people don't read it anyway. there's constantly issues opened claiming proxychains doesn't work when they test it with `ping` even though the very first paragraph makes it clear that only TCP is supported...

**Comment by 0XC7R** [comment]
> i'll probably add some detail to the README next time i work on it. though it seems most people don't read it anyway. there's constantly issues opened claiming proxychains doesn't work when they test it with `ping` even though the very first paragraph makes it clear that only TCP is supported...

"Ask dumb questions get no answers" thats the way I see it. 

Also you could add a header 1 to the very very top acting as a notice and if they do not listen to it you can just close the posts which will save you a bit of time, they're unable to ignore it then :wink: .

Been busy sorry for the late reply celebrated something. Also had to repair a randomly corrupted HDD volume so fun.

**Comment by 0XC7R** [comment]
> > i'll probably add some detail to the README next time i work on it. though it seems most people don't read it anyway. there's constantly issues opened claiming proxychains doesn't work when they test it with `ping` even though the very first paragraph makes it clear that only TCP is supported...
> 
> "Ask dumb questions get no answers" thats the way I see it.
> 
> Also you could add a header 1 to the very very top acting as a notice and if they do not listen to it you can just close the posts which will save you a bit of time, they're unable to ignore it then 😉 .
> 
> Been busy sorry for the late reply celebrated something. Also had to repair a randomly corrupted HDD volume so fun.

One thing I forgot to mention is issue templates. I have my own set on my own github project (Quick-Short if you want to copy the template). You can use these to your advantage by slapping a warning inside about TCP only or whatever else.
